release/nightly_tests/shuffle/shuffle_compute_large_scale.yaml

```diff
@@ -9,12 +9,12 @@ aws:
 
 head_node_type:
     name: head_node
-    instance_type:  i3.8xlarge
-    resources: {"object_store_memory": 21474836480}
+    instance_type:  m5.4xlarge
+    resources: {"object_store_memory": 21474836480, "cpu": 0}
```

Comment:
Hmm why cpu:0? It might not be reflective of user cluster configurations

Reply:
I think it's more stable for larger partitions and benchmarking but we can remove it for nightly tests.